### ISSUE TITLE
Experience with AOCL BLAS/LAPACK and Flexiblas

### ISSUE THREAD

**Issue by maximilianbehr** [mpimd-csc/flexiblas#47]
Dear @grisuthedragon, 

I wanted to ask about your experience with the AOCL Libraries offered by AMD and flexiblas. Is this setup tested on a regular basis?

**Comment by grisuthedragon** [comment]
Not yet. You mean AOCL-BLIS?

**Comment by maximilianbehr** [comment]
Yes https://www.amd.com/en/developer/aocl.html - it ships with blis and flame.

**Comment by Micket** [comment]
I stumbled over this old ticket. I use AOCL-BLAS as the default backend for my zen4 nodes (university cluster, tons of software, so very well tested in production)

**Comment by grisuthedragon** [comment]
As long as the AOCL-BLIS shared object is linked correctly, i.e. it references the aocl utils library, everything is fine.
